tmt/utils.py

```diff
@@ -257,13 +260,14 @@ def debug(self, key, value=None, color=None, shift=1, level=1, err=False):
 
     def _run(
         self, command, cwd, shell, env, log, join=False, interactive=False,
-        timeout=None):
+        timeout=None, select_timeout=DEFAULT_SELECT_TIMEOUT):
         """
         Run command, capture the output
 
         By default stdout and stderr are captured separately.
         Use join=True to merge stderr into stdout.
         Use timeout=<seconds> to finish process after given time
+        Use select_timeout=<seconds> to tweak select(timout) if necessary
```

Comment:
typo: `timout`

Reply:
In the end we don't export this via arguments.